README.md

```diff
@@ -1,135 +1,333 @@
-| :------------------------------------------------------------------------------------------ | :------------------------------------------------- |
-| a customer signs up for first time                                                          | /customer to create it                             |
-| a customer profile is updated                                                               | /customer to update it                             |
-| a new category is added to a product                                                        | /product to update its categories                  |
-| a new product is created                                                                    | /product to create it                              |
-| a new order is made                                                                         | /order to create it                                |
-| an order's status or details are changed                                                    | /order to update it                                |
-| a customer doesn't sign up, but makes an order just by filling in his email(guest customer) | /customer and then /order to create both resources |
-
-
-<br/><br/>
-## Responses from the endpoints
-The following responses are valid for all of the endpoints.
-
-| Code | Message |  
-| --- | --- |  
-| 204 | Success response. |  
-| 400 | Bad request from client. |  
-| 401 | Project token is invalid. |  
-| 402 | Project requires payment. |  
-| 403 | The IP that the request came from is ignored. |  
-| 500 | Error from server while processing request. |  
-| 502 | Error from server accepting request. |  
+:warning: **This guide is intended to be used by developers. If you're not a developer - please find one and come back later :)**
+
+Metrilo helps you track the events that the people visiting your shop do - viewing specific pages or products, adding products to the cart, making orders, etc. The provided data is transformed into different reports and analytics metrics you could benefit from.
+
+For more information about Metrilo and its features please visit [the official website](https://www.metrilo.com/).
+
+## Overview
+
+:warning: **We strongly recommend doing this on a Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
+
+[Quickstart guide](#quickstart-guide)
+
+[API reference](#api-reference)
+
+[Importing resources](#importing-resources)
+
+[Managing resources](#creating-and-updating-resources)
+
+[Tracking library installation](#tracking-library-installation)
+
+[Tracking events](#tracking-events)
+
+### Quickstart guide
+
+:warning: If you decide to follow along, you'll end up with some resources that you (most probably) won't use in the future. However, that's perfectly fine - after you're done testing, you can just ping us and we'll reset your project to a pristine state.
```

Comment:
There are lots of warning icons and I'm afraid to use this tutorial. Can we lower the stress from the whole document a bit?

Reply:
The only other suitable emoji I found is that one - ℹ️ - I think I can replace most of the warnings with it - would that work?

Reply:
Yeah, better.